docs/src/pages/components/text-fields/FormattedInputs.js

```diff
@@ -16,15 +16,24 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
-function TextMaskCustom(props) {
-  const { inputRef, ...other } = props;
+const TextMaskCustom = React.forwardRef(function TextMaskCustom(props, ref) {
+  const setRef = React.useCallback(
+    (maskedInputRef) => {
+      const value = maskedInputRef?.inputElement ?? null;
```

Comment:
```suggestion
      const value = maskedInputRef ? maskedInputRef.inputElement : null
```

to keep the old behavior. There a a couple of edge cases that neither approach covers so let's keep the old behavior to not accidentally break something.